### ISSUE TITLE
SVG not being rendered

### ISSUE THREAD

**Issue by ThomasLandauer** [PhenX/php-svg-lib#6]
Following up on [this issue at dompdf](https://github.com/dompdf/dompdf/issues/320#issuecomment-243382813):

Here's an example SVG which is not rendering:
`<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="419" height="200"> <path stroke-linejoin="round" stroke-linecap="round" stroke-width="5" stroke="rgb(100, 100, 100)" fill="rgb(255, 0, 0)" d="M 1 52 c 0.51 -0.14 19.57 -6.07 29 -8 c 3.13 -0.64 6.71 0.39 10 0 c 8.01 -0.94 15.97 -3.22 24 -4 c 5.61 -0.55 11.37 -0.24 17 0 c 2.02 0.09 5.72 0.08 6 1 c 0.43 1.43 -1.48 7.48 -3 9 c -1.13 1.13 -4.79 1 -7 1 c -1.59 0 -4.74 -0.03 -5 -1 c -0.9 -3.43 -0.62 -12.38 0 -18 c 0.33 -2.96 1.14 -7.14 3 -9 c 4.52 -4.52 12.37 -10.14 19 -13 c 9.57 -4.13 21.42 -8.04 32 -9 c 21.32 -1.94 47.04 -1.06 67 0 c 2.66 0.14 5.45 2.51 8 4 c 1.43 0.83 2.87 1.87 4 3 c 1.13 1.13 2.77 2.63 3 4 c 0.64 3.84 -0.14 13.29 0 14 c 0.05 0.23 1.13 -2.99 2 -4 c 0.97 -1.13 2.58 -2.56 4 -3 c 3.45 -1.08 7.97 -1.86 12 -2 c 15.58 -0.53 31.85 -0.77 47 0 c 3.99 0.2 8.17 1.62 12 3 c 4.43 1.6 8.7 4.41 13 6 c 1.83 0.67 4.17 0.39 6 1 c 2.02 0.67 4.31 1.74 6 3 c 2.13 1.6 5.2 4.09 6 6 c 0.57 1.37 -0.21 4.74 -1 6 c -0.58 0.93 -2.64 1.55 -4 2 c -1.55 0.52 -3.33 0.93 -5 1 c -5.9 0.26 -12.2 0.26 -18 0 c -1.33 -0.06 -4.07 -0.49 -4 -1 c 0.14 -0.97 3.04 -5.81 5 -6 c 14.32 -1.39 38.43 -1.44 57 0 c 10.99 0.85 21.88 4.17 33 7 c 8.96 2.28 17.34 5.03 26 8 c 3.14 1.08 6.18 2.48 9 4 c 1.43 0.77 3.45 1.9 4 3 c 0.55 1.1 0.49 3.66 0 5 c -0.7 1.93 -2.27 5.87 -4 6 c -18.23 1.37 -74.25 -0.12 -76 0 c -0.45 0.03 17.03 3.14 19 4 c 0.43 0.19 -1.89 2.39 -3 3 c -1.65 0.92 -4 1.86 -6 2 c -11.4 0.81 -23.51 0.94 -36 1 c -58.63 0.29 -111.3 0.59 -169 0 c -8.9 -0.09 -17.26 -2 -26 -2 c -12.09 0 -23.61 1.65 -36 2 c -11.69 0.33 -24.34 -0.52 -34 0 c -1.02 0.06 -2 1.75 -3 2 c -1.38 0.34 -5.09 0 -5 0 c 4.51 0 258 0 258 0"/> </svg>`

In base64:
`PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIgd2lkdGg9IjQxOSIgaGVpZ2h0PSIyMDAiPg0KICAgIDxwYXRoIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHN0cm9rZS1saW5lY2FwPSJyb3VuZCIgc3Ryb2tlLXdpZHRoPSI1IiBzdHJva2U9InJnYigxMDAsIDEwMCwgMTAwKSIgZmlsbD0icmdiKDI1NSwgMCwgMCkiIGQ9Ik0gMSA1MiBjIDAuNTEgLTAuMTQgMTkuNTcgLTYuMDcgMjkgLTggYyAzLjEzIC0wLjY0IDYuNzEgMC4zOSAxMCAwIGMgOC4wMSAtMC45NCAxNS45NyAtMy4yMiAyNCAtNCBjIDUuNjEgLTAuNTUgMTEuMzcgLTAuMjQgMTcgMCBjIDIuMDIgMC4wOSA1LjcyIDAuMDggNiAxIGMgMC40MyAxLjQzIC0xLjQ4IDcuNDggLTMgOSBjIC0xLjEzIDEuMTMgLTQuNzkgMSAtNyAxIGMgLTEuNTkgMCAtNC43NCAtMC4wMyAtNSAtMSBjIC0wLjkgLTMuNDMgLTAuNjIgLTEyLjM4IDAgLTE4IGMgMC4zMyAtMi45NiAxLjE0IC03LjE0IDMgLTkgYyA0LjUyIC00LjUyIDEyLjM3IC0xMC4xNCAxOSAtMTMgYyA5LjU3IC00LjEzIDIxLjQyIC04LjA0IDMyIC05IGMgMjEuMzIgLTEuOTQgNDcuMDQgLTEuMDYgNjcgMCBjIDIuNjYgMC4xNCA1LjQ1IDIuNTEgOCA0IGMgMS40MyAwLjgzIDIuODcgMS44NyA0IDMgYyAxLjEzIDEuMTMgMi43NyAyLjYzIDMgNCBjIDAuNjQgMy44NCAtMC4xNCAxMy4yOSAwIDE0IGMgMC4wNSAwLjIzIDEuMTMgLTIuOTkgMiAtNCBjIDAuOTcgLTEuMTMgMi41OCAtMi41NiA0IC0zIGMgMy40NSAtMS4wOCA3Ljk3IC0xLjg2IDEyIC0yIGMgMTUuNTggLTAuNTMgMzEuODUgLTAuNzcgNDcgMCBjIDMuOTkgMC4yIDguMTcgMS42MiAxMiAzIGMgNC40MyAxLjYgOC43IDQuNDEgMTMgNiBjIDEuODMgMC42NyA0LjE3IDAuMzkgNiAxIGMgMi4wMiAwLjY3IDQuMzEgMS43NCA2IDMgYyAyLjEzIDEuNiA1LjIgNC4wOSA2IDYgYyAwLjU3IDEuMzcgLTAuMjEgNC43NCAtMSA2IGMgLTAuNTggMC45MyAtMi42NCAxLjU1IC00IDIgYyAtMS41NSAwLjUyIC0zLjMzIDAuOTMgLTUgMSBjIC01LjkgMC4yNiAtMTIuMiAwLjI2IC0xOCAwIGMgLTEuMzMgLTAuMDYgLTQuMDcgLTAuNDkgLTQgLTEgYyAwLjE0IC0wLjk3IDMuMDQgLTUuODEgNSAtNiBjIDE0LjMyIC0xLjM5IDM4LjQzIC0xLjQ0IDU3IDAgYyAxMC45OSAwLjg1IDIxLjg4IDQuMTcgMzMgNyBjIDguOTYgMi4yOCAxNy4zNCA1LjAzIDI2IDggYyAzLjE0IDEuMDggNi4xOCAyLjQ4IDkgNCBjIDEuNDMgMC43NyAzLjQ1IDEuOSA0IDMgYyAwLjU1IDEuMSAwLjQ5IDMuNjYgMCA1IGMgLTAuNyAxLjkzIC0yLjI3IDUuODcgLTQgNiBjIC0xOC4yMyAxLjM3IC03NC4yNSAtMC4xMiAtNzYgMCBjIC0wLjQ1IDAuMDMgMTcuMDMgMy4xNCAxOSA0IGMgMC40MyAwLjE5IC0xLjg5IDIuMzkgLTMgMyBjIC0xLjY1IDAuOTIgLTQgMS44NiAtNiAyIGMgLTExLjQgMC44MSAtMjMuNTEgMC45NCAtMzYgMSBjIC01OC42MyAwLjI5IC0xMTEuMyAwLjU5IC0xNjkgMCBjIC04LjkgLTAuMDkgLTE3LjI2IC0yIC0yNiAtMiBjIC0xMi4wOSAwIC0yMy42MSAxLjY1IC0zNiAyIGMgLTExLjY5IDAuMzMgLTI0LjM0IC0wLjUyIC0zNCAwIGMgLTEuMDIgMC4wNiAtMiAxLjc1IC0zIDIgYyAtMS4zOCAwLjM0IC01LjA5IDAgLTUgMCBjIDQuNTEgMCAyNTggMCAyNTggMCIvPg0KPC9zdmc+`

Outcome: The image-area is being rendered (as I can see from a border I added), but it's blank. Only when I added the fill (as above) I get so see at least something. However, it has the wrong color ;-) More important to me: The stroke is not rendered at all.

Bottom line: I really need this functionality in a few weeks. But it's no big deal if I use another image format. So would you guess that this will be working soon, or should I switch to PNG right away?

Thanks!!

**Comment by PhenX** [comment]
Ok, please tell me if that's OK now if you replace the line 176 in Svg/Style.php by this one : 

        $parts = preg_split('/[^,]\s+/', $color, 2);

**Comment by ThomasLandauer** [comment]
Yeah, thanks, awesome!! :-)

However, it was line 128 for me. For anybody else: Search for `$parts = preg_split('/\s+/', $color, 2);`

**Comment by Asimov500** [comment]
I have tried loading svgs with the img tag and I always get "image not found or type unknown". Can you explain what you did to get yours working?
